avocado/utils/genio.py

```diff
@@ -10,7 +10,7 @@
 #
 # This code was inspired in the autotest project,
 # client/shared/utils.py
-# Authors: Martin J Bligh <mbligh@google.com>, Andy Whitcroft <apw@shadowen.org>
+# Authors:Martin J Bligh <mbligh@google.com>, Andy Whitcroft<apw@shadowen.org>
```

Comment:
Do you have the link to the failure? I never saw this failing (and the space actually makes sense there)